images/gitlab-delete-stale-projects/gitlab-cleanup.py

```diff
@@ -179,10 +182,14 @@ def notify(project, group_id):
         logger.debug("Notifications not configured")
         return False
     
-    requests.post(
-        notification_url, headers={"x-notification-token": notification_token, 'Content-type': 'application/json'},
-        json={"event_name": "project_deleted" if project is not None else "group_deleted", "project": project, "group_id": group_id}
-    )
+    try:
+        resp = requests.post(
```

Comment:
Would it be good to honor the `dry_run` option here as well so the notification isn't sent if it's just a dry run?

Reply:
Ok. Added notification change. Now won't send notifications on dry run unless then token is `DRYRUN`